tools/accuracy_checker/openvino/tools/accuracy_checker/launcher/opencv_launcher.py

```diff
@@ -73,7 +73,8 @@ def parameters(cls):
                 regex=BACKEND_REGEX, choices=OpenCVLauncher.OPENCV_BACKENDS.keys(),
                 optional=True, default='IE',
                 description="Backend name: {}".format(', '.join(OpenCVLauncher.OPENCV_BACKENDS.keys()))),
-            'inputs': ListInputsField(optional=False, description="Inputs.")
+            'inputs': ListInputsField(optional=False, description="Inputs."),
+            'allow_reshape_input': BoolField(optional=True, default=False, description="Allows reshape input.")
```

Comment:
this key means that instead of resizing data to fit model, model should be reshaped to data side. Does opencv support model reshaping or dynamic model inference?